src/ktopt_planning_context.cpp

```diff
@@ -248,33 +249,113 @@ void KTOptPlanningContext::transcribePlanningScene(const planning_scene::Plannin
+        {
+          const auto objectptr = std::dynamic_pointer_cast<const shapes::Box>(shape);
+          const SourceId box_source_id = scene_graph.RegisterSource(shape_name);
+          const GeometryId box_geom_id = scene_graph.RegisterAnchoredGeometry(
+              box_source_id,
+              std::make_unique<GeometryInstance>(
+                  RigidTransformd(pose),
+                  std::make_unique<Box>(objectptr->size[0], objectptr->size[1], objectptr->size[2]), shape_name));
+
+          // add illustration, proximity, perception properties
+          scene_graph.AssignRole(box_source_id, box_geom_id, IllustrationProperties());
+          scene_graph.AssignRole(box_source_id, box_geom_id, ProximityProperties());
+          scene_graph.AssignRole(box_source_id, box_geom_id, PerceptionProperties());
+          break;
+        }
+        case shapes::ShapeType::UNKNOWN_SHAPE:
+        {
+          RCLCPP_WARN(getLogger(), "Unknown shape, ignoring in scene graph");
+          break;
+        }
+        case shapes::ShapeType::SPHERE:
+        {
+          const auto objectptr = std::dynamic_pointer_cast<const shapes::Sphere>(shape);
+          const SourceId box_source_id = scene_graph.RegisterSource(shape_name);
+          const GeometryId box_geom_id = scene_graph.RegisterAnchoredGeometry(
+              box_source_id, std::make_unique<GeometryInstance>(
+                                 RigidTransformd(pose), std::make_unique<Sphere>(objectptr->radius), shape_name));
+          RCLCPP_INFO(getLogger(), "Sphere");
+
+          // add illustration, proximity, perception properties
+          scene_graph.AssignRole(box_source_id, box_geom_id, IllustrationProperties());
+          scene_graph.AssignRole(box_source_id, box_geom_id, ProximityProperties());
+          scene_graph.AssignRole(box_source_id, box_geom_id, PerceptionProperties());
+          break;
+        }
+        case shapes::ShapeType::CYLINDER:
+        {
+          const auto objectptr = std::dynamic_pointer_cast<const shapes::Cylinder>(shape);
+          const SourceId box_source_id = scene_graph.RegisterSource(shape_name);
+          const GeometryId box_geom_id = scene_graph.RegisterAnchoredGeometry(
+              box_source_id,
+              std::make_unique<GeometryInstance>(
+                  RigidTransformd(pose), std::make_unique<Cylinder>(objectptr->radius, objectptr->length), shape_name));
+          RCLCPP_INFO(getLogger(), "Cylinder");
+
+          // add illustration, proximity, perception properties
+          scene_graph.AssignRole(box_source_id, box_geom_id, IllustrationProperties());
+          scene_graph.AssignRole(box_source_id, box_geom_id, ProximityProperties());
+          scene_graph.AssignRole(box_source_id, box_geom_id, PerceptionProperties());
+          break;
+        }
+        case shapes::ShapeType::CONE:
+        {
+          RCLCPP_WARN(getLogger(), "Cone not supported in drake");
+          break;
+        }
```

Comment:
Are these all possible shape types? If not, you should add a default like this at the end of the switch statement
```
default:
{
  RCLCPP_WARN(getLogger(), "Shape TYPE conversation to drake is not implemented");
}
```